graphdatascience/query_runner/protocol/project_protocols.py

```diff
@@ -31,12 +33,16 @@ def run_projection(
 
     @staticmethod
     def select(protocol_version: ProtocolVersion) -> "ProjectProtocol":
```

Comment:
you dont need to use the string version if you import `from __future__ import annotations` at the top.